tests/test.yml

```diff
@@ -2,5 +2,6 @@
 - hosts: localhost
   remote_user: root
   roles:
-    - role: ansible-role-virtualenvwrapper
-      virtualenvwrapper_shell_rc_file: /home/travis/.bashrc
\ No newline at end of file
+    # Current parameterized role, look at '../defaults/main.yml' to see all variables
+    - role: ../..
+      virtualenvwrapper_shell_rc_file: /home/travis/.bashrc
```

Comment:
@marconius I forgot to ask you something. Can you upgrade the version of the role please?? You can do it on a new pull request 
ref: https://github.com/gcporras/ansible-role-virtualenvwrapper/blob/master/meta/main.yml